directory-connector/js/MainController.js

```diff
@@ -11,6 +11,11 @@ Ext.define('Ung.apps.directory-connector.MainController', {
     getSettings: function () {
         var me = this, v = this.getView(), vm = this.getViewModel();
 
+        // set the enabled custom block page based on URL existance
+        vm.bind('{settings.radiusSettings.server}', function (url) {
+            vm.set('settings.radiusSettings.enabled', url.length > 0);
+        });
+
```

Comment:
I don't think we should be removing the checkbox for this.  I'm ok with us writing an invalid ip/hostname if the user has the checkbox disabled.  We'll never use the ip value in the backend

Reply:
Changed to method in other PR